### ISSUE TITLE
2.x version?

### ISSUE THREAD

**Issue by joakim** [ItsEeleeya/tauri-plugin-trafficlights-positioner#2]
Would be neat to have this as a plugin for Tauri 2, now that it's out. Do you plan to make a 2.x version?

Edit: I just found [clearlysid/tauri-plugin-decorum](https://github.com/clearlysid/tauri-plugin-decorum/) which is for Tauri 2. That should solve the issue for me, but I'll let you answer and close in case you have any plans.

**Comment by ItsEeleeya** [comment]
Decorum, Yes! I was originally thinking about that but I didn't see that much point in doing it.
It's better to just implement the delegates for your app and have full control.
Plugin Decorum is still missing some stuff I needed/wanted. It still doesn't persist the traffic lights location on resize.
I have a [PR there](https://github.com/clearlysid/tauri-plugin-decorum/pull/24) that adds/fixes some of the issues with it.
I'm still not certain if I should focus on just macOS or have a universal plugin.
Or maybe have something like [Tauri Controls](github.com/agmmnn/tauri-controls/) but as a full plugin.
For now I think I'll only try fixing the main issues with Plugin Decorum as the rest of the stuff seems to be better left for each app to decide.

**Comment by joakim** [comment]
Yeah, I hit the resize issues with Decorum on macOS. Notably with [tauri_plugin_window_state](https://github.com/tauri-apps/tauri-plugin-window-state), which resizes windows as soon as they're created.

Your PR looks great though! I think I'll wait for that to go in.

Tauri Controls _looks_ good, but it's not native. That's a no-go for me, I need full native functionality for the traffic lights.

**Comment by ItsEeleeya** [comment]
Thank you,
I think I'll end up writing a gist as a template for doing that. Probably with _Tauri Controls_ but with the native controls on macOS and the snaps layout on Windows 11 (Without a hotkey hack).
VSCode and Firefox seem to have their own trick because they can easily move the location of those on macOS. (for VSCode it seems like they did a lot more than what Electron provides)
